src/ui/right_panel.rs

```diff
@@ -97,7 +97,7 @@ fn scene_explorer(
 
                     let is_visible = visibility
                         .get(entity)
-                        .map(|v| v.1 == Visibility::Visible)
+                        .map(|v| v.1 != Visibility::Hidden)
```

Comment:
It's out of scope for this PR, but a simple fix: initially the visibility is at "inherited", and the parent is always visible, so we can safely consider `inherited` as visible.